index.d.ts

```diff
@@ -31,12 +32,16 @@ export class Client {
 }
 
 export class Frame {
-  constructor(command: string, headers?: {}, body?: string);
+  command: string;
+  body: string;
+  headers: ExtendedHeaders;
 
+  constructor(command: string, headers?: {}, body?: string);
   toString(): string;
-  sizeOfUTF8(s: string): number;
-  unmarshall(datas: any): any;
-  marshall(command: string, headers?: {}, body?: string): any;
+
+  static unmarshallSingle(data: string): Frame;
+  static unmarshall(datas: string): { frames: Frame[], partial?: string };
+  static marshall(command: string, headers?: {}, body?: string): string;
```

Comment:
Declare `headers` as `ExtendedHeaders` for the same reason as explained for the `constructor`.